src/tsolvers/bvsolver/Bvector.h

```diff
@@ -87,7 +87,7 @@ class Bvector {
         header.size      = 0;
     }
 
-    Bvector    operator=   (Bvector)         { assert(false); return *this; }
+    Bvector&    operator=   (Bvector const &) = delete;
```

Comment:
@aehyvari , Do you know what is the purpose of this `Bvector` class? I have not tried, but from the search in IDE it looks like it is not used anywhere...

Reply:
Yes it's a structure that holds the bits of a gate. I use it in the counting code that maybe goes to master in a relatively short time frame. Maybe we can keep it for now if it's not a big problem.

Reply:
OK!